docs/src/main/sphinx/release/release-453.md

```diff
@@ -138,3 +138,8 @@
 ## SQL Server connector
 
 * Add support for the `execute` procedure. ({issue}`22556`)
+
+## SPI
+
+* Add `SystemAccessControl.getTableColumnMasks` as replacement for the
```

Comment:
I don't think #21997 added `getTableColumnMasks` method. https://github.com/trinodb/trino/pull/21997/files#diff-9c1c468145e70554a262a95c50bd9bdd474935e69663e803df567939f0b4da0cR967

Reply:
Ah, yes, the PR originally had it as getTableColumnMasks but we renamed it before merging. Good catch!

Reply:
Nice! Adjusted to that now and made the commit message longer ;-)